querybook/webapp/components/Search/SearchResultItem.tsx

```diff
@@ -47,6 +52,93 @@ function openClick(
     }
 }
 
+interface IQueryItemProps {
+    preview: IQueryPreview;
+    searchString: string;
+    environmentName: string;
+}
+
+export const QueryItem: React.FunctionComponent<IQueryItemProps> = ({
+    preview,
+    environmentName,
+    searchString,
+}) => {
+    const {
+        author_uid: authorUid,
+        created_at: createdAt,
+        engine_id: engineId,
+        id,
+        query_text: queryText,
+        title,
+    } = preview;
+    const { userInfo: authorInfo, loading } = useUser({ uid: authorUid });
+
+    const isQueryCell = preview.query_type === 'query_cell';
+
+    const url = isQueryCell
+        ? `/${environmentName}/datadoc/${preview.data_doc_id}/?cellId=${id}`
+        : `/${environmentName}/query_execution/${id}/`;
+    const handleClick = React.useMemo(() => openClick.bind(null, url), [url]);
```

Comment:
`useCallback(() => openClick(url), [url])`

Reply:
Added event to function call as well to make sure it handles command click correctly